src/Dto/GameDto.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Dto;
+
+use App\Entity\Game;
+
+class GameDto
```

Comment:
I'm not sure I understand what this dto is for ... the `get` methods were not public ?

Reply:
If I don't use something like a srializer, the response will be empty : `{}`